torch/_inductor/ir.py

```diff
@@ -6745,7 +6745,7 @@ def extract_scan_args(
         _, fx_init, fx_xs, _, _, fx_additional_inputs = extract_scan_args(
             *V.graph.current_node.args
         )
-        fx_all_inputs = fx_init + fx_xs + fx_additional_inputs
+        fx_all_inputs = fx_init + fx_xs + tuple(fx_additional_inputs) if type(fx_init) == tuple else fx_additional_inputs
```

Comment:
```suggestion
        fx_all_inputs = fx_init + fx_xs + tuple(fx_additional_inputs) if type(fx_init) is tuple else fx_additional_inputs
```

Reply:
This has been resolved in a different way